source/atlas-cli-deploy-local.txt

```diff
@@ -330,6 +333,36 @@ deployment.
       #. Specify how you want to connect to the deployment and press 
          ``Enter``.
 
+   .. step:: Pause a deployment.
+
+      a. Run the following command to pause a deployment:
+
+         .. code-block:: sh
+
+            atlas deployments pause
+
+      #. Specify the deployment to pause and press ``Enter``.
```

Comment:
I usually use the :kbd: directive for the Enter key but not sure what the style guide says on this? Applies to other cases on this page as well
```suggestion
      #. Specify the deployment to pause and press :kbd:`Enter`.
```